kvbc/src/kvbc_app_filter/kvbc_app_filter.cpp

```diff
@@ -263,8 +264,12 @@ void KvbAppFilter::readBlockRange(BlockId block_id_start,
                                   spsc_queue<KvbFilteredUpdate> &queue_out,
                                   const std::atomic_bool &stop_execution) {
   if (block_id_start > block_id_end || block_id_end > rostorage_->getLastBlockId()) {
+    if (block_id_start <= block_id_end) {
+      num_storage_reads++;
+    }
     throw InvalidBlockRange(block_id_start, block_id_end);
   }
+  num_storage_reads++;
 
```

Comment:
Why do we increment twice only? Shouldn't we increment with `block_id_end - block_id_start` ?